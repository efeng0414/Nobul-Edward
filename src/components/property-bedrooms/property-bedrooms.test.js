import React from "react";
import renderer from "react-test-renderer";
import PropertyBedrooms from "./index";
import { IntlProvider } from "react-intl";

describe("PropertyBedrooms", () => {
  const componentProps = {
    onBedroomsChange: () => {},
    selectedBedrooms: 0,
    intl: global.getIntl(),
    jobType: "test"
  };

  const componentJSX = (
    <IntlProvider {...componentProps}>
      <PropertyBedrooms />
    </IntlProvider>
  );

  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
