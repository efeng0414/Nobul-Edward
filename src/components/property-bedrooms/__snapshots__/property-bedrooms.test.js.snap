// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyBedrooms Component Snapshot should match existing snapshot 1`] = `
<div
  className="property-bedrooms"
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="property-bedrooms-parent"
  >
    <img
      className="property-bedrooms-icon"
      src={
        Object {
          "default": "file-mock",
        }
      }
    />
    <div
      className="property-bedrooms-container-undefined"
    >
      <div
        className="ant-input-number property-bedrooms-input"
        style={Object {}}
      >
        <div
          className="ant-input-number-handler-wrap"
        >
          <span
            aria-disabled={false}
            aria-label="Increase Value"
            className="ant-input-number-handler ant-input-number-handler-up "
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchCancel={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            unselectable="unselectable"
          >
            <span
              className="ant-input-number-handler-up-inner"
              onClick={[Function]}
              unselectable="unselectable"
            />
          </span>
          <span
            aria-disabled={true}
            aria-label="Decrease Value"
            className="ant-input-number-handler ant-input-number-handler-down ant-input-number-handler-down-disabled"
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            role="button"
            unselectable="unselectable"
          >
            <span
              className="ant-input-number-handler-down-inner"
              onClick={[Function]}
              unselectable="unselectable"
            />
          </span>
        </div>
        <div
          aria-valuemax={99}
          aria-valuemin={0}
          aria-valuenow={0}
          className="ant-input-number-input-wrap"
          role="spinbutton"
        >
          <input
            autoComplete="off"
            className="ant-input-number-input"
            max={99}
            min={0}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseUp={[Function]}
            required={false}
            step={1}
            value="0"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
